apollo-router/src/plugins/include_query_plan.rs

```diff
@@ -0,0 +1,241 @@
+use futures::future::ready;
+use futures::stream::once;
+use futures::StreamExt;
+use http::HeaderValue;
+use serde_json_bytes::json;
+use tower::util::BoxService;
+use tower::BoxError;
+use tower::ServiceExt as TowerServiceExt;
+
+use crate::layers::ServiceExt;
+use crate::plugin::Plugin;
+use crate::plugin::PluginInit;
+use crate::register_plugin;
+use crate::services::QueryPlannerContent;
+use crate::services::QueryPlannerRequest;
+use crate::services::QueryPlannerResponse;
+use crate::services::RouterRequest;
+use crate::services::RouterResponse;
+
+const INCLUDE_QUERY_PLAN_HEADER_NAME: &str = "X-Apollo-Query-Plan";
```

Comment:
The `X-` prefixes are deprecated these days, so I think we can just go with something without `X-`?